topics/ui/controls/combo_box.md

```diff
@@ -23,79 +23,86 @@ comboBox.setEditable(true);
 
 ## When to use
 
-Use a combo box if:
+Use a combo-box if:
 
-* The user needs to select a value or enter a custom value.
+* The user needs to select a value or enter a custom value:
 
-  ![](combo_box_font_size.png){width=165}
+  ![A font size combo-box that allows selecting predefined values as well as entering a custom value](combobox_when_to_use_1.png){width=706}
 
 * The possible choices are objects or states.
 
 
 * The user may need to return to previous values. For example, save previously entered paths, so the user can quickly
   select them later:
 
-  ![](maven.png){width=413}
+  ![A combo-box with an open menu with previously selected values](combobox_when_to_use_2.png){width=706}
 
 ### When not to use
 
-If the number of options is finite, and there’s no need to enter custom values. In this case, use a [drop-down list](drop_down.md) or [radio buttons](radio_button.md).
+* If the number of options is finite, and there is no need to enter custom values. In this case, use a [drop-down list](drop_down.md) or [radio buttons](radio_button.md):
 
-If it’s not possible to list the most likely choices. In this case, use an [input field](input_field.md) instead.
+  ![The Theme drop-down with a predefined list of values](combobox_when_not_to_use_1.png){width="706"}
 
-![](prefill.png){width=152}
+* If it is not possible to list the most likely choices. In this case, use an [input field](input_field.md) instead:
+  
+  ![A port number input](combobox_when_not_to_use_2.png){width=706}
 
-If the list is big, and the user knows what value they need and won’t review the list. In this case, use an input field with completion.
-
-![](input_field_completion.png){width=509}
+* If the list is big, and the user knows what value they need and won’t review the list. In this case, use an input field with completion:
+  
+  ![An input field with a completion popup](combobox_when_not_to_use_3.png){width=706}
 
 ## How to use
 
-For the [label](drop_down.md#label) and the [default value](drop_down.md#default-value) apply the same rules as for the
+* For the [label](drop_down.md#label) and the [default value](drop_down.md#default-value) apply the same rules as for the
 [drop-down list](drop_down.md).
```

Comment:
Same about separate sections instead of bullet points

Reply:
Done. Could you please check the headings?

Reply:
Very nice, thank you!